tests/e2e/kcs/test_noobaa_rebuild.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import (
+    ignore_leftovers,
+    E2ETest,
+    tier3,
+    skipif_openshift_dedicated,
+    skipif_ocs_version,
+    skipif_external_mode,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.constants import DEFAULT_NOOBAA_BUCKETCLASS, DEFAULT_NOOBAA_BACKINGSTORE
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_noobaa_pods
+from ocs_ci.ocs.resources.pvc import get_pvc_objs
+
+logger = logging.getLogger(__name__)
+
+
+@tier3
+@ignore_leftovers
+@pytest.mark.polarion_id("OCS-2653")
+@pytest.mark.bugzilla("1991361")
+@skipif_ocs_version("<4.9")
+@skipif_openshift_dedicated
+@skipif_external_mode
+class TestNoobaaRebuild(E2ETest):
+    """
+    Test to verify noobaa rebuild.
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.noobaa_op = "noobaa-operator"
```

Comment:
This can be replaced with constant NOOBAA_OPERATOR in defaults

Reply:
replaced